src/main/scala/com/github/unchama/seichiassist/data/HalloweenItemData.scala

```diff
@@ -79,18 +79,18 @@ object HalloweenItemData {
   //region HalloweenHoe
 
   val halloweenHoe: ItemStack = {
-    val displayName = List(
-        s"${RED}C", s"${GOLD}E", s"${YELLOW}N", s"${GREEN}T", s"${BLUE}E", s"${DARK_AQUA}O", s"${LIGHT_PURPLE}T", s"${RED}L")
-        .map(str => s"$BOLD$ITALIC$str")
-        .mkString
+    val displayName = Seq(
+      (RED, "C"), (GOLD, "E"), (YELLOW, "N"), (GREEN, "T"), (BLUE, "E"), (DARK_AQUA, "O"), (LIGHT_PURPLE, "T"), (RED, "L"))
+      .map(name => s"${name._1}$BOLD$ITALIC${name._2}")
```

Comment:
suggestion: destructure
```suggestion
      .map { case (color, c) => s"$color$BOLD$ITALIC$c" }
```